docs/identity/saas-apps/veza-tutorial.md

```diff
@@ -78,39 +79,24 @@ Follow these steps to enable Microsoft Entra SSO.
 
 1. On the **Basic SAML Configuration** section, perform the following steps:
 
-	a. In the **Identifier** text box, type a value using one of the following patterns:
+	a. In the **Identifier** text box, enter the Veza Audience URI (Entity ID), for example `https://<customer>.vezacloud.com/auth/saml/metadata`:
 
-    | **Identifier** |
-    |-------------|
-    `urn:auth0:<Cookie-auth0-instance-name>:saml-<customer-name>-cookie-connection` |
-    `urn:auth0:<Veza-auth0-instance-name>:saml-<customer-name>-cookie-connection` |
+	b. In the **Reply URL** text box, enter the Veza Single sign-on URL (ACS), for example `https://<customer>.vezacloud.com/auth/saml/acs`.
 
-    b. In the **Reply URL** text box, type a URL using one of the following patterns:
-
-    | **Reply URL** |
-    |---------|
-    | `https://<instancename>.veza.com` |
-    | `https://<instancename>.cookie.ai` |
-
-1. Click **Set additional URLs** and perform the following step if you wish to configure the application in **SP** initiated mode:
-	
-    In the **Sign-on URL** text box, type a URL using one of the following patterns:
+1. In the **Sign-on URL** text box, type a URL using one of the following patterns if you wish to configure the application in **SP** initiated mode:
```

Comment:
Adjusted these instructions to match the Azure UI. However, I'm not sure how users can retrieve this from Veza -- can we derive it from the Veza `Audience URI` or `ACS` URL?